### PR TITLE
updated readme and added contributing file

### PR DESCRIPTION
**Pull request by tjsavage**
@sjmiles @arthurevans for sanity check on the component examples and content

### CODE REVIEW

**Comment by tjsavage** [comment]
@kevinpschaaf as well for the original request

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Filing bugs
+
+The Polymer team heavily uses (and loves!) Github for all of our software management. We use Github issues to track all bugs and features.
+
+If you find an issue, please do file it on the repository. The [Polymer/polymer issues](https://github.com/polymer/polymer/issues) should be used only for issues on the Polymer library itself - bugs somewhere in the core codebase.
+
+For issues with elements the team maintains, please file directly on the element's repository. If you're not sure if a bug stems from the element or the library, air toward filing it on the element and we'll move the issue if necessary.
+
+We love examples for addressing issues - issues with a Plunkr, jsFiddle, or jsBin will be much easier for us to work on quickly.
```
Comment (kevinpschaaf) [inline CONTRIBUTING.md]: Suggest http://jsbin.com/cijadomupa/1/edit?html,output as a starting point?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+* [Twitter](https://twitter.com/polymer)
+* [Google+ Community](https://plus.sandbox.google.com/u/0/communities/115626364525706131031?cfem=1)
+* [Mailing list](https://groups.google.com/forum/#!forum/polymer-dev)
+* [Slack channel](bit.ly/polymerslack)
+
+## The Polymer Repositories
+
+Because of the component-based nature of the Polymer project, we tend to have lots of different repositories. Our main repository for the Polymer library itself is at [github.com/Polymer/polymer](https://github.com/polymer/polymer). File any issues or pull requests that have to do with the core library on that repository, and we'll take a look ASAP.
+
+We keep all of our element "product line" repos in the [PolymerElements](https://github.com/polymerelements) organization. For any element-specific issues or pull requests, file directly on the element's repository, such as the `paper-button` repository at [github.com/polymerelements/paper-button](https://github.com/polymerelements/paper-button).
```
Comment (kevinpschaaf) [inline CONTRIBUTING.md]: Suggest clarifying "all our elements" to "elements the Polymer team maintains and distributes"... and can we give a nod to community catalogs where other Polymer elements can be found?

File: CONTRIBUTING.md (hunk 1 above)
Comment (tjsavage) [inline CONTRIBUTING.md]: perfect I'll add that, that's really helpful.

File: CONTRIBUTING.md (hunk 2 above)
Comment (tjsavage) [inline CONTRIBUTING.md]: Agreed, good point. Will update.

**Pull request merged**